packages/svelte/src/compiler/phases/2-analyze/visitors/shared/a11y.js

```diff
@@ -991,12 +991,31 @@ export function check_element(node, state) {
 
 	// element-specific checks
 	let contains_a11y_label = false;
+	let missing_content_warning_fired = false;
```

Comment:
I hold no love for this. Happy to think on alternatives... But if it's fine, I won't lose sleep.

Reply:
What is it for? We don't deduplicate other warnings, we just let them rip

Reply:
nvm, figured it out — it's to prevent duplication with the `a11y_missing_content` warning. Simpler to just remove `'a'` from the list of elements that are relevant to that warning, so that's what I did